kyo-core/shared/src/main/scala/kyo/KyoApp.scala

```diff
@@ -55,9 +46,6 @@ object KyoApp:
             initCode += (() => handle(v))
     end Base
 
-    /** The combined effect type used by KyoApp. */
-    type Effects = Async & Resource & Abort[Throwable]
```

Comment:
I think this indirection can be confusing to users. Splicing the type union on each use will allow users to directly see what effects are supported by `run` for example.